todo/src/reducers/index.js

```diff
@@ -0,0 +1,51 @@
+import {
+  ADD_TODO,
+  TOGGLE_COMPLETED,
+  DELETE_TODO,
+  REMOVE_COMPLETED
+} from "../actions";
+
+const initialState = {
+  todos: [
+    {
```

Comment:
I like that you populated the data to let the user know how to interact w/ the data.